src/main/play-doc/operation/CLI.md

```diff
@@ -90,52 +90,6 @@ shazar ./visualizer-poll-interval.sh
 
 The configuration is now ready to be loaded along with a bundle. Configuration is always provided as the second param to the `conduct load` command.
 
-## Deploying bundles
```

Comment:
I'm wondering if this section should be retained, with a separate section on orchestrating bundle deployment. WDYT? Have you been through the flow reading from the CLI section, to the next section etc?